climpred/utils.py

```diff
@@ -29,11 +32,14 @@ def get_metric_function(metric, dict_):
     if isinstance(metric, types.FunctionType):
         return metric
     else:
-        is_in_dict(metric, dict_, 'metric')
-        return dict_[metric]
+        # equivalent of: `if metric in METRIC_ALIASES;
+        # METRIC_ALIASES[metric]; else metric`
+        is_in_list(metric, list_, 'metric')
+        metric = METRIC_ALIASES.get(metric, metric)
```

Comment:
Okay, so this line should be switched with the one above it. Since this line will retrieve the proper metric if it's an alias ('pr' --> 'pearson_r'), then tests with `is_in_list` after if it's an actual metric.